### PR TITLE
completed all happy path and error handling tests for post to comment…

### PR DESCRIPTION
**Pull request by Jordan96L**
…s by review id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -250,5 +250,100 @@ describe("my express app", () => {
...
+          return request(app)
+            .post("/api/reviews/3/comments")
+            .send(badBodyWithMissingKey)
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toBe("The information provided was incorrect");
+            });
+        });
+      });
+    });
```
Comment (karlaferry) [inline __tests__/app.test.js:357]: I would also add a test for when the post body is empty!

**Review by karlaferry — APPROVED** [review]
Approving but add one more test then merge :)
